roman.py

```diff
@@ -33,21 +33,21 @@ def get_roman_digit(number, base):
     if quotient >= 5:
         # do something including the half base
         if quotient == 9:
-            sys.stdout.write('%s%s' % (roman_prefix[(base*10)], roman_digit[(base*10)])) 
+            sys.stdout.write('%s%s' % (ROMAN_PREFIX[(base*10)], ROMAN_DIGIT[(base*10)])) 
         else:
-            sys.stdout.write('%s' % roman_digit[(base*5)])
+            sys.stdout.write('%s' % ROMAN_DIGIT[(base*5)])
             remaining = quotient - 5
-            sys.stdout.write('%s' % roman_digit[base] * remaining)
+            sys.stdout.write('%s' % ROMAN_DIGIT[base] * remaining)
```

Comment:
Funny, upon first viewing, i thought both methods you sent were the same (apples to apples). But upon close inspection it isn't.  

Actually, thanks for the tip. I do prefer your way better, much more perl-ish.  I really care not to have to tell the damned string what type i'm passing the info.